kwok/main.go

```diff
@@ -41,7 +43,7 @@ func main() {
 	ctx, op := operator.NewOperator()
 	instanceTypes, err := kwok.ConstructInstanceTypes()
 	if err != nil {
-		logging.FromContext(ctx).Fatalf("failed to construct instance types: %s", err)
+		log.FromContext(ctx).Error(fmt.Errorf("constructing instance type, %w", err), "Startup error")
```

Comment:
@jonathan-innis let's de-capitalize it here as well :) ?

Reply:
I just synced with @njtran. The upstream guidance is to use capatalization with messages. You can see this all throughout client-go and all throughout upstream k8s binaries.

Reply:
There are still some left apparently. Let me fix those quickly.

Reply:
Updated the capitalization to adhere to upstream convention around capitalizing the message but keeping error messages as lowercase.

Reply:
:+1:

Reply:
I take it back. I like @njtran's suggestion. Just undid what I did. Apologize for the churn.